shellcheck.yml

```diff
@@ -0,0 +1,43 @@
+name: Shellcheck Lint
+
+on:
+  push:
+    paths:
```

Comment:
```suggestion

```
I would remove that. I'm happy if it would run on all paths and all branches.

Reply:
This won't work. We need this instead:
```
on:
  push:
    branches:
      # Run workflow on every push
      - '**'        # matches every branch

  pull_request:
    branches:
      # Run workflow on every push
      - '**'        # matches every branch
```